### PR TITLE
Enable pkg/sentry/hostcpu support on arm64.

### PR DESCRIPTION
**Pull request by xiaobo55x**
Signed-off-by: Haibo Xu haibo.xu@arm.com
Change-Id: I333872da9bdf56ddfa8ab2f034dfc1f36a7d3132

### CODE REVIEW

**Comment by xiaobo55x** [comment]
Ref #63

**Comment by xiaobo55x** [comment]
Hi @prattmic,

Please help review this patch!

Thanks,
Haibo

File: pkg/sentry/hostcpu/getcpu_arm64.s [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "textflag.h"
+
+// GetCPU makes the getcpu(unsigned *cpu, unsigned *node, NULL) syscall
+// for the lake of optimazing way to get the current CPU number on arm64.
```
Comment (prattmic) [inline pkg/sentry/hostcpu/getcpu_arm64.s]: "... lack of an optimized way of getting .."

Comment (xiaobo55x) [reply pkg/sentry/hostcpu/getcpu_arm64.s]: Fixed, thanks!

**Comment by xiaobo55x** [comment]
@prattmic, Thanks!

**Pull request merged**